Uno.Gallery/Uno.Gallery.Shared/Views/Shell.xaml

```diff
@@ -20,9 +208,186 @@
 		</Grid.RowDefinitions>
 
 		<!-- Banner similar to the Uno website (WASM only) -->
-		<wasm:Border Grid.Row="1">
-			<local:Banner />
-		</wasm:Border>
+		<wasm:Grid x:Name="RootGrid"
+				   Grid.Row="1"
+				   Padding="32,16">
+			<VisualStateManager.VisualStateGroups>
+				<VisualStateGroup>
+					<VisualState x:Name="Desktop">
+						<VisualState.StateTriggers>
+							<AdaptiveTrigger MinWindowWidth="1000" />
```

Comment:
threshold should be the same as the nav-view, which is exactly `DesktopAdaptiveThresholdWidth`

Reply:
also, instead of toggling inner children's visibility, you could just do RootGrid's